File: CS/WindowsApplication3/Main.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Windows.Forms;
using DevExpress.XtraEditors;

namespace DxExample {
    public partial class Main : XtraForm {
        OfficeNavigationBarHelper helper;
        public Main() {
            InitializeComponent();
            officeNavigationBar1.PeekFormShowDelay = 100;
            helper = new OfficeNavigationBarHelper(officeNavigationBar1);
            helper.SetPeekFormBehavior(true);
        }
     
        private void officeNavigationBar1_QueryPeekFormContent(object sender, DevExpress.XtraBars.Navigation.QueryPeekFormContentEventArgs e) {
            if(e.Item.Text == "Item1")
                e.Control = new XtraUserControl() { BackColor = Color.Green};
            if (e.Item.Text == "Item2")
                e.Control = new XtraUserControl() { BackColor = Color.DeepSkyBlue };
        }
        protected override void OnFormClosing(FormClosingEventArgs e) {
            helper.SetPeekFormBehavior(false);
            base.OnFormClosing(e);
        }
    }
}

File: VB/WindowsApplication3/Main.vb
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq
Imports System.Windows.Forms
Imports DevExpress.XtraEditors

Namespace DxExample
    Partial Public Class Main
        Inherits XtraForm

        Private helper As OfficeNavigationBarHelper
        Public Sub New()
            InitializeComponent()
            officeNavigationBar1.PeekFormShowDelay = 100
            helper = New OfficeNavigationBarHelper(officeNavigationBar1)
            helper.SetPeekFormBehavior(True)
        End Sub

        Private Sub officeNavigationBar1_QueryPeekFormContent(ByVal sender As Object, ByVal e As DevExpress.XtraBars.Navigation.QueryPeekFormContentEventArgs) Handles officeNavigationBar1.QueryPeekFormContent
            If e.Item.Text = "Item1" Then
                e.Control = New XtraUserControl() With {.BackColor = Color.Green}
            End If
            If e.Item.Text = "Item2" Then
                e.Control = New XtraUserControl() With {.BackColor = Color.DeepSkyBlue}
            End If
        End Sub
        Protected Overrides Sub OnFormClosing(ByVal e As FormClosingEventArgs)
            helper.SetPeekFormBehavior(False)
            MyBase.OnFormClosing(e)
        End Sub
    End Class
End Namespace

